appsvc_fnc_dev_teamslink/GetTeamsLink.cs

```diff
@@ -34,44 +30,63 @@ public static async Task Run([TimerTrigger("0 0 10-21/2 * * 1-5")] TimerInfo myT
 
             Auth auth = new Auth();
             var graphClient = auth.graphAuth(log);
-            var UpdateList = new ListItemsCollectionPage();
-            List<CreateItem> CreateList = new List<CreateItem>();
 
-            var queryOptions = new List<QueryOption>()
+            List<CreateItem> CreateList = new();
+            List<Microsoft.Graph.Models.Group> groups = new();
+            List<ListItem> items = new();
+            List<ListItem> UpdateList = new();
+
+            ////////////////////////////////////////////////////////////////////////////////////////////////////////////
+            // Get items from TeamsLink list                                                                          //
+            ////////////////////////////////////////////////////////////////////////////////////////////////////////////
+            var listitems = await graphClient.Sites[siteId].Lists[listId].Items.GetAsync((requestConfiguration) =>
             {
-                new QueryOption("expand", "fields(select=TeamsID,Teamslink)")
-            };
-            List<ListItem> items = new List<ListItem>();
-            var AllTeamsItems = await graphClient.Sites[siteId].Lists[listId].Items
-            .Request(queryOptions)
-            .Top(999)
-            .GetAsync();
-
-            items.AddRange(AllTeamsItems.CurrentPage.OfType<ListItem>());
-            // fetch next page
-            while (AllTeamsItems.NextPageRequest != null)
+                requestConfiguration.QueryParameters.Expand = new string[] { "fields($select=TeamsID,Teamslink)" };
+                requestConfiguration.QueryParameters.Top = 10;  // 999
```

Comment:
Is this not suppose to be 999?

Reply:
Yes, I used 10 for testing and forgot to revert to 999. Fixed.